modules/theme/src/definitions/views/admin-portal.less

```diff
@@ -115,54 +115,54 @@
 }
 
 // VARIABLES
-@super-treeview-text-color: #000;
-@super-treeview-arrow-color: #000;
-@super-treeview-tree-stem-color: #000;
+@treeview-tree-stem-color: #f3f3f3;
```

Comment:
Move variables to variable file

Reply:
fixed with commit 8d0673e7d9a2fca1bb84b137ada06c5569e79369